src/aks-preview/HISTORY.md

```diff
@@ -2,6 +2,10 @@
 
 Release History
 ===============
+0.4.21
++++++
+* Support cluster certificate rotation operation using "az aks rotate-certs"
```

Comment:
Could you also add this line : Add support for `az aks kanalyze`

Reply:
done